vue.config.js

```diff
@@ -13,4 +13,27 @@ module.exports = {
       ],
     },
   },
-}
+  devServer: {
+    before(app) {
+      // Mock a route to mimic this upload result format:
+      // {"audio_file_id":147,"duration_seconds":"4.803250","description":"xyz.wav","created_at":{"date":"2020-11-24 01:41:58","timezone_type":3,"timezone":"UTC"},"audio_uuid":"5fbc64e0c74e90.82972899"}"
+      app.all('/audiofiles/upload', (req, res) => {
```

Comment:
Hi hi, how does this work from the consumer repository's perspective? Will this be active from viamo platform even though the route is already implemented?

Reply:
As discussed, we will add doc in readme to help devs activating the right route during integration